start-client/src/components/common/error/Boot.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+class Boot extends React.Component {
+  render = () => {
+    return (
+      <div className='control-error'>
+        <p className='title'>
+          <strong>
+            The version {this.props.value} of Spring Boot is outdated.
+          </strong>
+        </p>
+        <p>
+          You have to select a valid version if you want to generate or explore
```

Comment:
How about "Select a valid version to generate or explore the project"?

This also suggests that you can still share a project with invalid configuration but trying to do so results in an error. I guess it should say something like "Select a valid version to generate, explore, or share the project"?

I also wonder if the buttons for generating, exploring, and sharing should change appearance somehow when the configuration is invalid such that they won't do anything.